infra/modules/service/variables.tf

```diff
@@ -70,9 +70,20 @@ variable "vpc_id" {
   description = "Uniquely identifies the VPC."
 }
 
-variable "subnet_ids" {
+variable "public_subnet_ids" {
```

Comment:
These changes were all pulled from the Nava template. Here's the diff between our version and main:

https://github.com/navapbc/template-infra/compare/fe5c7cd24d3c2c9f15c342826cda0a20af4cd0a5...main